src/openrct2/paint/Supports.h

```diff
@@ -94,6 +94,19 @@ enum class MetalSupportType : uint8_t
     BoxedCoated,
 };
 
+enum class MetalSupportPlace : uint8_t
+{
+    Corner0 = 0,
+    Corner1 = 1,
+    Corner2 = 2,
+    Corner3 = 3,
+    Centre = 4,
+    Side0 = 5,
+    Side1 = 6,
+    Side2 = 7,
+    Side3 = 8,
+};
```

Comment:
I'm guessing these either mean on corner of the or the centre of one side, right? So I think a better naming would be something like:

```suggestion
    TopRightCorner = 0,
    BottomLeftCorner = 1,
    BottomRightCorner = 2,
    TopRightCorner = 3,
    Centre = 4,
    CentreRight = 5,
    CentreBottom = 6,
    CentreLeft = 7,
    CentreTop = 8,
};
```

(Just guessed the directions, but I'm assuming this would be more meaningful)

Reply:
Right, I will look into that. I’ll also have to check if they’re ordinal (left/right/etc.) or cardinal (north/south) directions.